services/ui-src/src/components/FormHeader/FormHeader.jsx

```diff
@@ -3,15 +3,15 @@ import { connect } from "react-redux";
 import { Link } from "react-router-dom";
 import PropTypes from "prop-types";
 import { Button, TextInput, Table } from "@trussworks/react-uswds";
-import { getFormTypes, getSingleForm } from "../../libs/api";
+import { getSingleForm } from "../../libs/api";
 import {
   updateFPL,
   saveForm
 } from "../../store/reducers/singleForm/singleForm";
 import { getUserInfo } from "../../utility-functions/userFunctions";
 
 const FormHeader = ({ quarter, form, year, state, updateFPL, saveForm }) => {
-  const [formDescription, setFormDescription] = useState({});
+  const formDescription = formData.find(element => element.form === form);
```

Comment:
Thank you and I'm sorry; this is a diligent reproduction of the error in my suggestion >_<

```suggestion
  const formDescription = formTypes.find(element => element.form === form);
```